### DISCUSSION TITLE
The time the object spent in the frame.

### DISCUSSION THREAD

**Discussion by epigraphe** [ultralytics/ultralytics#5824] [Q&A] [answered]
Good day. Tell me, please, is it possible to somehow calculate the time an object is in the frame? That is, while the conditional person is in the area, the time of his presence is counted. It is advisable to show the time within the frame of the object. Like that
![image](https://github.com/ultralytics/ultralytics/assets/6083395/5afb85a8-cbfc-4460-9f26-183de679227f)
That is, there are 6 jobs in the video and I need to count how much time a person spends on it. That is, I will make 6 zones, and as soon as there is an object in the zone, the time counter starts. The object has left the zone - there is no counter. He came in again - the counter went again.
As I understand that I will need object tracking, I’m thinking about bytetrack.
But I can't figure out what to do next.

I'm currently set up following this guide.
https://blog.roboflow.com/how-to-count-objects-in-a-zone/

**Comment by epigraphe** [comment, marked as answer]
The issue is resolved. I took the time the object entered the frame, the time the object exited the frame, and the timer from the entry comment. Then I just output it to [labels]
Нolo is not needed here, everything is done in pure python

**Reply by pderrenger** [reply]
@epigraphe great to hear that you've resolved the issue! It sounds like you've implemented a custom solution to track the time an object spends in a frame, which is a good approach for your specific use case. If you have any more questions or need further assistance with YOLOv8 in the future, feel free to reach out. Happy coding! 😊👍
